app/views/categories/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="flex flex-col h-full">
+  <div class="flex flex-row w-[vw] bg-[#3778c2] p-3 text-white items-center justify-center">
+    <p class="flex flex-row place-self-center">CATEGORIES</p>
+  </div>
+
+  <div>
+    <% @categories.each do |category| %>
+      <%= link_to category_deals_path(category) do %>
+  <div class="border-solid border-2 p-2 m-1 flex flex-row" id="<%= dom_id category %>" class="">
+  <%= link_to category_deals_path(category), class: "flex flex-row justify-between w-full" do %>
+    <%= image_tag(category.icon, size: "80x80", alt: "Group") %>
+
+    <div class="flex flex-col justify-between">
+      <p>
+        Category:
+        <strong><%= category.name %></strong>
+      </p>
+
+      <p>
+        Total Transactions:
+        <strong><%= category.deals.length %></strong>
+      </p>
```

Comment:
- [ ] According to the project requirements, in the categories page you are required to list "their name, icon and the total amount of all the transactions that belongs to that category". Currently the total amounts for the category is not being listed instead the number of categories is. Kindly make this change.

![image](https://user-images.githubusercontent.com/86318284/208646042-258f2276-c491-458f-868b-0f401fa76a7d.png)

**Example of how the Categories page looks like on my end:**

![image](https://user-images.githubusercontent.com/86318284/208646171-3534c04a-8d20-49a0-97d8-382c824e29aa.png)

**Sample example of how it is supposed to look like (note the amounts on the side of each category)**

![image](https://user-images.githubusercontent.com/86318284/208646369-a9e1a083-60f7-4a88-9053-99f968114531.png)